CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.1]
```

Comment:
We need the links at the bottom of the doc updated as well, reference here: https://github.com/RedHatInsights/topological_inventory-providers-common/commit/792b61f6cb7b3b6af6e4b180847c36a51dbbb83f

Reply:
Missing the date next to 2.0.1, I know 2.0.0 didn't have it but it needs it.

Reply:
@lindgrenj6 I'm usually adding the date to previous version, because it's not clear when exactly this PR/gem version will be released. Maybe we should talk about it and set some rule